reference-implementation/lib/transform-stream.js

```diff
@@ -150,28 +131,10 @@ function TransformStreamEnqueueToReadable(transformStream, chunk) {
   }
 }
 
-function TransformStreamError(transformStream, e) {
-  if (transformStream._errored === true) {
-    throw new TypeError('TransformStream is already errored');
-  }
-
-  TransformStreamErrorInternal(transformStream, e);
-}
-
-function TransformStreamErrorIfNeeded(transformStream, e) {
-  if (transformStream._errored === false) {
-    TransformStreamErrorInternal(transformStream, e);
-  }
-}
-
+// This is a no-op if both sides are already errored.
 function TransformStreamErrorInternal(transformStream, e) {
```

Comment:
This should probably lose the -Internal suffix now?

Reply:
That makes sense.